### PR TITLE
Added the Skedulo API guide to docsearch json config file

### PR DESCRIPTION
**Pull request by gsheldon**
We use ReDoc for our API documentation, located under `/skedulo-api/` on our developer site. It would be great if DocSearch could also search this page.

### CODE REVIEW

**Comment by gsheldon** [comment]
I'm not 100% certain I have added the selectors for `skedulo-api` correctly. This part of the docs is generated using ReDoc, which provides some interesting code when it renders the page.

File: configs/skedulo_developer.json [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
+      "lvl0": ".td-content h1",
+      "lvl1": ".td-content h2",
+      "lvl2": ".td-content h3",
+      "lvl3": ".td-content h4",
+      "lvl4": ".td-content h5",
+      "lvl5": ".td-content h6",
+      "text": ".td-content p, .td-content li"
+    },
+    "skedulo-api": {
+      "lvl0": ".WxWXp h1",
```
Comment (s-pace) [inline configs/skedulo_developer.json]:
```suggestion
      "lvl0": ".api-content h1",
      "lvl1": ".api-content h2",
      "lvl2": ".api-content h3",
      "lvl3": ".api-content h4",
      "lvl4": ".api-content h5",
      "text": ".redoc-markdown p, .redoc-markdown li"
```

File: configs/skedulo_developer.json [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
+      "lvl1": ".ioYTqA h2",
```
Comment (s-pace) [inline configs/skedulo_developer.json]:
```suggestion
```

File: configs/skedulo_developer.json [hunk 3]
```diff
@@ -1,20 +1,28 @@
...
+      "text": ".flfxUM p, .SmuWE span"
```
Comment (s-pace) [inline configs/skedulo_developer.json]:
```suggestion
```

File: configs/skedulo_developer.json [hunk 4]
```diff
@@ -1,20 +1,28 @@
...
   "stop_urls": [],
   "selectors": {
-    "lvl0": ".td-content h1",
-    "lvl1": ".td-content h2",
-    "lvl2": ".td-content h3",
-    "lvl3": ".td-content h4",
-    "lvl4": ".td-content h5",
-    "lvl5": ".td-content h6",
-    "text": ".td-content p, .td-content li"
+    "docs": {
```
Comment (s-pace) [inline configs/skedulo_developer.json]:
```suggestion
    "default": {
```

File: configs/skedulo_developer.json [hunk 5]
```diff
@@ -1,20 +1,28 @@
 {
   "index_name": "skedulo_developer",
   "start_urls": [
-    "https://developer.skedulo.com/docs/"
+    "https://developer.skedulo.com/docs/",
+    "https://developer.skedulo.com/skedulo-api/"
```
Comment (s-pace) [inline configs/skedulo_developer.json]:
```suggestion
{
      "url": "https://developer.skedulo.com/skedulo-api/",
      "selectors_key": "skedulo-api"
}
```

**Pull request merged**